src/Manipulations.php

```diff
@@ -59,7 +59,14 @@ class Manipulations
 
     public function __construct(array $manipulations = [])
     {
-        $this->manipulationSequence = new ManipulationSequence($manipulations);
+        //Check if is a single array or multi arrays
+        if ($this->checkIfHasMultipleConversions($manipulations)) {
+            foreach ($manipulations as $manipulation) {
+                $this->manipulationSequence = new ManipulationSequence($manipulation);
+            }
+        } else {
```

Comment:
This `else` statement can be avoided by wrapping `$manipulations` in an extra array if it does not contain multiple conversions.